javascript-modal/main.js

```diff
@@ -0,0 +1,18 @@
+var $buttonClick = document.querySelector('.button');
+var $closeButton = document.querySelector('.button.close');
+var $modalBox = document.querySelector('.modal-box');
+
+function click(event) {
+  if ($buttonClick.className === 'button open') {
+    $modalBox.className = 'modal-box on';
+  }
```

Comment:
If you are more specific about which `.button` when you select it, you can lose the `if` statement.